composer.json

```diff
@@ -12,22 +12,23 @@
         "mdm",
         "demo"
     ],
-	"require" : {
-		"docopt/docopt": "^1.0"
-	},
-	"require-dev" : {
-		"squizlabs/php_codesniffer" : "^2.7",
-		"glpi-project/coding-standard" : "0.5",
-		"glpi-project/tools": "^0.1.2",
-		"phpunit/phpunit" : "^4.8 || ^5.4 || ^6.0",
-		"flyve-mdm/test" : "dev-master",
-		"consolidation/robo" : "^1.0",
+    "require" : {
+        "docopt/docopt": "^1.0",
+        "gregwar/captcha": "^1.1"
+    },
+    "require-dev" : {
+        "squizlabs/php_codesniffer" : "^2.7",
+        "glpi-project/coding-standard" : "0.5",
+        "glpi-project/tools": "^0.1.2",
+        "phpunit/phpunit" : "^4.8 || ^5.4 || ^6.0",
+        "flyve-mdm/test" : "dev-master",
+        "consolidation/robo" : "^1.0",
         "evert/phpdoc-md": "^0.2.0",
         "phpdocumentor/phpdocumentor": "2.*"
-	},
-	"repositories" : [{
-			"type" : "vcs",
-			"url" : "https://github.com/flyve-mdm/flyve-mdm-glpi-test"
-		}
-	]
-}
\ No newline at end of file
+    },
+    "repositories" : [{
+            "type" : "vcs",
+            "url" : "https://github.com/flyve-mdm/flyve-mdm-glpi-test"
```

Comment:
.git is not necessary to clone the repo, and doing this way makes easier for someone to pick the url and browse to the project.